api/src/main/resources/dao/categorical-attribute-dao.xml

```diff
@@ -32,17 +32,45 @@
                 ]]>
             </value>
         </property>
+        <property name="loadAllAttributesValuesWithoutLinksQuery">
+            <value>
+                <![CDATA[
+                    SELECT key, value, id FROM pipeline.categorical_attributes
+                ]]>
+            </value>
+        </property>
         <property name="loadAllAttributesValuesQuery">
             <value>
                 <![CDATA[
-                    SELECT key, value FROM pipeline.categorical_attributes
+                    SELECT
+                        attributes.key,
+                        attributes.value,
+                        attributes.id,
+                        links.child_id,
+                        links.autofill,
+                        links_values.key child_key,
+                        links_values.value as child_value
+                    FROM pipeline.categorical_attributes attributes
+                    LEFT OUTER JOIN pipeline.categorical_attributes_links links on attributes.id=links.parent_id
```

Comment:
I think we can omit `OUTER` in requests, since `LEFT JOIN` is always `LEFT OUTER JOIN`